mlflow/models/dependencies_schemas.py

```diff
@@ -66,18 +70,18 @@ def _get_retriever_schema():
     Returns:
         VectorSearchIndex: The vector search index schema.
     """
-    if not globals().get(DATABRICKS_RETRIEVER_PRIMARY_KEY, None) or not globals().get(
-        DATABRICKS_RETRIEVER_TEXT_COLUMN, None
+    if not globals().get(_RETRIEVER_PRIMARY_KEY, None) or not globals().get(
+        _RETRIEVER_TEXT_COLUMN, None
     ):
         return []
 
     return [
         RetrieverSchema(
-            name="retriever",
-            primary_key=globals().get(DATABRICKS_RETRIEVER_PRIMARY_KEY, None),
-            text_column=globals().get(DATABRICKS_RETRIEVER_TEXT_COLUMN, None),
-            doc_uri=globals().get(DATABRICKS_RETRIEVER_DOC_URI, None),
-            other_columns=globals().get(DATABRICKS_RETRIEVER_OTHER_COLUMNS, None),
+            name=globals().get(_RETRIEVER_NAME, None),
+            primary_key=globals().get(_RETRIEVER_PRIMARY_KEY, None),
+            text_column=globals().get(_RETRIEVER_TEXT_COLUMN, None),
+            doc_uri=globals().get(_RETRIEVER_DOC_URI, None),
+            other_columns=globals().get(_RETRIEVER_OTHER_COLUMNS, None),
```

Comment:
do we need to call `globals()` once?

Reply:
This code has not changed since 2.13.0, I can do this as a follow-up change with some manual testing. I want to make sure nothing breaks in end-to-end testing